apps/frontend/src/components/form/NextButton.tsx

```diff
@@ -13,9 +13,10 @@ const NextButton: React.FC<NextButtonProps> = ({
 }) => {
   return (
     <Button
-      colorScheme="teal"
+      bg="#EA6824"
+      _hover={{ bg: "#EA682461" }}
```

Comment:
Lets either get rid of this or make it a brighter orange instead of a lighter one. Right now the button looks grayed out/disabled when you hover.